bootstrap.d/dev-util.yml

```diff
@@ -150,6 +150,35 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: ninja
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: The Ninja build system
+      description: Ninja is a small build system similar to make.
+      spdx: 'Apache-2.0'
+      website: 'https://ninja-build.org/'
+      maintainer: "Dennis Bonke <dennis@managarm.org>"
+      categories: ['dev-util']
+    source:
+      subdir: ports
+      git: 'https://github.com/ninja-build/ninja.git'
+      tag: 'v1.10.2'
+      version: '1.10.2'
+    tools_required:
+      - system-gcc
+      - host-cmake
+    configure:
+      - args:
+        - 'cmake'
+        - '-DCMAKE_TOOLCHAIN_FILE=@SOURCE_ROOT@/scripts/CMakeToolchain-@OPTION:arch-triple@.txt'
+        - '@THIS_SOURCE_DIR@'
+    build:
+      - args: ['make', '-j@PARALLELISM@']
+      - args: ['mkdir', '-p', '@THIS_COLLECT_DIR@/usr/bin']
+      - args: ['cp', 'ninja', '@THIS_COLLECT_DIR@/usr/bin']
```

Comment:
Is there no 'make install' rule?

Reply:
I just checked on a local clone with `make install -n` and it seems to do *something*, I think it should exist. Can you try?

Reply:
[Gentoo seems to agree.](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-util/ninja/ninja-9999.ebuild?id=f2ba94c014da24115072b0fdfd2ddea74c8ed81e#n104)

Reply:
@64 is right here, make install seems to work as intended. Switched to that, thanks for catching it.